legal-api/src/legal_api/models/furnishing.py

```diff
@@ -77,9 +77,12 @@ class FurnishingStatus(BaseEnum):
     # parent keys
     batch_id = db.Column('batch_id', db.Integer, db.ForeignKey('batches.id'), index=True, nullable=False)
     business_id = db.Column('business_id', db.Integer, db.ForeignKey('businesses.id'), index=True, nullable=False)
+    furnishing_group_id = db.Column('furnishing_group_id', db.Integer, db.ForeignKey('furnishing_groups.id'),
+                                    index=True, nullable=False)
```

Comment:
so we don't actually have an index on this column yet but I think it makes sense to.  

Can you add an index for this column in the alembic script?

Reply:
Added